src/s3_plugin.js

```diff
@@ -40,8 +41,9 @@ module.exports = class S3Plugin {
       s3Options = {},
       cdnizerOptions = {},
       s3UploadOptions = {},
-      cloudfrontInvalidateOptions = {}
-    } = options
+      cloudfrontInvalidateOptions = {},
+      git = {}
```

Comment:
I think the option could be simplified to `useGitHash` or something similar. Not sure this really needs the full object